pool.go

```diff
@@ -217,6 +298,15 @@ func (p *pool) fetchBlockWith(ctx context.Context, c cid.Cid, with string) (blk
 		if recordIfContextErr(resourceTypeBlock, ctx, "fetchBlockWithLoop") {
 			return nil, ctx.Err()
 		}
+
+		// sample request for mirroring
+		if p.config.MirrorFraction > rand.Float64() {
```

Comment:
We need a default value for `MirrorFraction`

Reply:
1% ?

Reply:
Sure, let's start with that !